collector/mountstats_linux.go

```diff
@@ -266,6 +286,129 @@ func NewMountStatsCollector(logger log.Logger) (Collector, error) {
 			nil,
 		),
 
+		NFSTransportReadChunkTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "transport_read_chunk_total"),
```

Comment:
Should probably be chunk*s*

Reply:
Thanks for your review. 
Done.
the counter counts be setted plural.